diff --git a/node_modules/@baronha/react-native-multiple-image-picker/android/build.gradle b/node_modules/@baronha/react-native-multiple-image-picker/android/build.gradle
index fcb659d..01c2341 100644
--- a/node_modules/@baronha/react-native-multiple-image-picker/android/build.gradle
+++ b/node_modules/@baronha/react-native-multiple-image-picker/android/build.gradle
@@ -1,6 +1,6 @@
 buildscript {
   // Buildscript is evaluated before everything else so we can't use getExtOrDefault
-  def kotlin_version = "1.4.32"
+  def kotlin_version = "1.6.0"
 
   repositories {
     google()
@@ -120,7 +120,7 @@ repositories {
   }
 }
 
-def kotlin_version = "1.4.32"
+def kotlin_version = "1.6.0"
 
 dependencies {
   // noinspection GradleDynamicVersion
